frontend/src/components/Accordion/Accordion.jsx

```diff
@@ -0,0 +1,44 @@
+import PropTypes from 'prop-types'
+import { useState } from 'react'
+import { UpIcon, DownIcon } from '../Icon/icons'
+import BodyText from '../BodyText/BodyText'
+import Headline from '../Headline/Headline'
+
+function Accordion({ className, headerText, fullText }) {
+  const [showFAQ, setShowFAQ] = useState(false)
+
+  return (
+    <div className={className}>
+      <div className="accordion-title-arrow">
```

Comment:
Can we change the `setShowFAQ(true)` function to be called here?

Reply:
I just added the changes